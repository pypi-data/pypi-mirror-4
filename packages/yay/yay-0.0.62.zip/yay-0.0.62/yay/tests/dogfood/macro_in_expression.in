
name: foo

.define SomeMacro:
    - SomeItem:
        name: ${name}
    - SomeOtherItem:
        name: ${name}


resources.append: ${SomeMacro!}


