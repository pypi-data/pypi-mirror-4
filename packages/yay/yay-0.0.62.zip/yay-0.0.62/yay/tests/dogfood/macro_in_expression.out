name: foo

resources:
    - SomeItem:
        name: foo
    - SomeOtherItem:
        name: foo

